rtv/terminal.py

```diff
@@ -552,7 +552,7 @@ def open_url_silent(url):
             with self.suspend():
                 webbrowser.open_new_tab(url)
 
-    def open_pager(self, data):
+    def open_pager(self, data, wrap=False):
```

Comment:
Just for convention's sake I think the default should be ``None`` instead of ``False``